tidb-cloud/tidb-cloud-encrypt-cmek.md

```diff
@@ -0,0 +1,118 @@
+                "Resource": "*",
+                "Condition": {
+                    "StringEquals": {
+                        "kms:CallerAccount": "<pingcap-account>",
+                        "kms:ViaService": "ec2.us-west-2.amazonaws.com"
+                    }
+                }
+            },
+            {
+                "Sid": "Allow TiDB cloud role to use KMS to store encrypted backup to S3",
+                "Effect": "Allow",
+                "Principal": {
+                    "AWS": "arn:aws:iam::<pingcap-account>:root"
+                },
+                "Action": [
+                    "kms:Decrypt",
+                    "kms:GenerateDataKey"
+                ],
+                "Resource": "*"
+            },
+            ... // user's own admin access to KMS
+        ]
+    }
+    ```
+
+> **Note:**
+>
+> - <pingcap-account> will be provided separately through documentation or by other means, which is the account where customer clusters run in.
+> - For EBS related policy in the first block, please refer to [aws doc](https://docs.aws.amazon.com/kms/latest/developerguide/conditions-kms.html#conditions-kms-caller-account) for reference.
+> - For S3 related policy in the second block, please refer to [aws blog](https://repost.aws/knowledge-center/s3-bucket-access-default-encryption) for reference.
+
+### Step 2. Create a new project and enable CMEK
+
+1. Using TiDB Cloud Open API, create a new project and enable AWS CMEK. 
+2. Configure the KMS Key ARN for the specified region (e.g., us-east-1) under this project using TiDB Cloud Open API.
+
+> **Note:**
+>
+> Currently the configuration is not available on UI, user can configure the project using TiDB cloud’s OpenAPI restful interface.
+
+### Step 3. Create a cluster
+
+Create an AWS Dedicated Cluster under the project created in step 1. You need to ensure that the region where the cluster is located is the same as that in step 2.
+
+> **Note:**
+>
+> When CMEK is enabled, the EBS volumes used by the nodes of the cluster and the S3 used for the cluster backups are encrypted using CMEK.
+
+## Rotate your CMEK
+
+You can configure [automatic CMEK rotation](http://docs.aws.amazon.com/kms/latest/developerguide/rotate-keys.html) on AWS KMS. AWS automatic CMEK rotation does not require you to update the TiDB Cloud Encryption at Rest project settings, including the CMEK ID.
+
+## Revoke and restore CMEK
+
+In the case PingCAP’s access to the CMEK needs to be temporarily revoked, the user needs to follow these steps:
+
+1. The user needs to revoke the corresponding permissions and update the KMS Key policy on the AWS KMS console.
+2. Pause all the clusters under the project on the TiDB Cloud console.
+
+> **Note:**
+>
+> If you are just revoking the CMEK on AWS KMS, this will not affect the running cluster. If you have revoked the CMEK, when you pause the cluster and then restore the cluster, the cluster will not be able to restore normally because it can not access the CMEK.
```

Comment:
```suggestion
> After you revoke CMEK on AWS KMS, your running clusters are not affected. However, when you pause a cluster and then restore the cluster, the cluster will not be able to restore normally because it cannot access CMEK.
```